cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -14,44 +17,75 @@ var (
 	since         int
 	contains      string
 	sortOrder     string
+	clusterID     string
 	namespaceList []string
 	nodeList      []string
 	podList       []string
 	containerList []string
 	statusList    []string
 )
 
+const (
+	logsCmdDescription = `
+  Fetch logs of current cluster context (by default) from Dynatrace and display the logs like oc logs.
+
+  This command also prints the Dynatrace URL and the corresponding DQL in the output.
+
+`
+
+	logsCmdExample = `
+  # Get the logs of HCP cluster in current cluster context.
+  $ osdctl cluster dynatrace logs
+
+  # Get the logs of a specific HCP cluster
+  $ osdctl cluster dynatrace logs -c <cluster-id>
+
+  # Get the logs of the pod alertmanager-main-0 in namespace openshift-monitoring
+  $ osdctl cluster dynatrace logs --po alertmanager-main-0 --namespace openshift-monitoring
+
+  # Only return logs newer than 2 hours old (an integer in hours)
+  $ osdctl cluster dynatrace logs --since 2
+
+  # Restrict return of logs to those that contain a specific phrase
+  $ osdctl cluster dynatrace logs --contains <phrase>
+`
+)
+
 func NewCmdLogs() *cobra.Command {
 	logsCmd := &cobra.Command{
-		Use:   "logs <cluster-id>",
-		Short: "Fetch logs from Dynatrace",
-		Long: `Fetch logs from Dynatrace and display the logs like oc logs.
-
-  This command also prints the Dynatrace URL and the corresponding DQL in the output.`,
-		Example: `
-  # Get the logs of HCP cluster hcp-cluster-id-123.
-  # Specify to get the logs of the pod alertmanager-main-0 in namespace openshift-monitoring
-  osdctl cluster dynatrace logs hcp-cluster-id-123 --namespace openshift-monitoring --pod alertmanager-main-0`,
-		Args:              cobra.ExactArgs(1),
+		Use:               "logs",
+		Short:             "Fetch logs from Dynatrace",
+		Long:              logsCmdDescription,
+		Example:           logsCmdExample,
+		Args:              cobra.NoArgs,
 		DisableAutoGenTag: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			err := main(args[0])
+			var err error
+			if clusterID == "" {
+				clusterID, err = GetClusterID()
+				if err != nil {
+					cmdutil.CheckErr(err)
+				}
+			}
+			err = main(clusterID)
 			if err != nil {
 				cmdutil.CheckErr(err)
 			}
 		},
 	}
 
+	logsCmd.Flags().StringVar(&clusterID, "cluster-id", "", "Cluster ID (default to current cluster context)")
```

Comment:
```suggestion
	logsCmd.Flags().StringVar(&clusterID, "cluster", "", "Name or ID of the cluster (defaults to current cluster context)")
```

Looking at other CLI tools like ROSA/OCM, we use `cluster` so would like to keep it similar
```
ocm list idps --cluster=mycluster
```
OR
```
rosa describe cluster --cluster=mycluster

Flags:
  -c, --cluster string             Name or ID of the cluster.
```